flathub.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
What about other architectures?

Reply:
As I mentioned above ARM uses different OpenGL backend and different hardware decoding, but I haven't implemented and tested it...